### ISSUE TITLE
Move voila-reveal to core voila

### ISSUE THREAD

**Issue by SylvainCorlay** [voila-dashboards/voila-reveal#22]
It is confusing to users that without installing `voila-reveal`, the command `voila --template=reveal` will work because the reveal template is found in nbconvert. Worse, it will simply fail silently to connect to the kernel because it is not a proper voilà template.

One way to make things better could be to move voila-reveal to core voila.

**Comment by SylvainCorlay** [comment]
@maartenbreddels what do you think of this proposal?

**Comment by maartenbreddels** [comment]
Yes, good call. Makes total sense, because reveal is standard in nbconvert as well.

**Comment by SylvainCorlay** [comment]
Thankfully, voila-reveal already specifies voila<0.3, so that we can add the template to core voilà and set the version to 0.3dev.
